Makefile

```diff
@@ -287,6 +293,7 @@ bundle-prepare: OPSDK generate kustomize ## Generate bundle manifests and metada
 .PHONY: bundle
 bundle: bundle-prepare
 	$(SED) -e 's/^/    /' config/manifests/bases/description-upstream.md > tmp-desc
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
```

Comment:
@jotak make bundle was broken the reason all ur testing with catalog was failing because the operator image u build and it has the webhook never reflected in CSV